docs/options.md

```diff
@@ -237,7 +260,7 @@ CIBW_TEST_COMMAND: nosetests {project}/tests
 CIBW_TEST_COMMAND: nosetests {project}/tests
 ```
 
-Platform-specific variants also available:  
```

Comment:
Sorry, missed this before. Could you also revert these spaces at the end of the line? Maybe we should do the same in a different PR as @mayeut already did elsewhere, replacing them with \ instead of , but it's nice to keep focused changesets.

Reply:
Given I'm copy-pasting this for the new `CIBW_REPAIR_COMMAND` I opted to fix it for all 7 instances in this PR (using `<br/>`, looks like `\` does not work for documents built with `mkdocs`).